### DISCUSSION TITLE
Come aggiungere Decdeal Termostato WiFi per Caldaia a Gas ad home assistant

### DISCUSSION THREAD

**Discussion by maurominella** [indomus/forum#1642] [HUB: Home Assistant]
Ciao ho acquistato [questo termostato](https://amzn.to/3uP0Uoz) -il più venduto su Amazon- per sostituire il precedente vecchio termostato.
Ho già le valvole dei termosifoni connesse ad home assistant tramite uno zigbee hub, ma questo termostato ha solo la connessione WiFi e peraltro si è connesso senza alcuna difficoltà, lo controllo tranquillamente con Alexa o Google Assistant, tramite Smart Life.

Io però voglio inserirlo in Home Assistant ma non ho capito da dove devo iniziare perchè a differenza dei dispositivi Tuya Zigbee non posso avviare un processo di discovery/accoppiamento. Esiste un integration service in cui posso specificare il suo indirizzo IP?

Grazie mille

**Comment by snakuzzo** [comment]
Se è associato tramite app Smartlife fa già parte dell'ecosistema Tuya. Ti basta abilitare l'integrazione Tuya e seguire le istruzioni [QUI](https://www.home-assistant.io/integrations/tuya/).
In alernativa, se volessi evitare il cloud ed usarlo completamente in locale, c'è il custom component [Local Tuya]( https://github.com/rospogrigio/localtuya)

**Reply by maurominella** [reply]
Ottimi spunti, grazie. Purtroppo vedo che la piattaforma climate non è ancora supportata ([la auspicavano oltre un anno fa](https://github.com/rospogrigio/localtuya/issues/439)), quindi mi resta solo l'opzione di installare l'integrazione Tuya, che ho provato e funziona.
Peccato per la mancanza del localtuya anche perché non vale esclusivamente per sistemi in locale, ma può essere usato anche se home assistant gira nel cloud come nel mio caso: basta infatti mappare la porta 8889 del router (UDP, non TCP) sull'IP del dispositivo, e si collega senza problemi. Infatti ho provato a simulare un altro device fra quelli supportati facendolo puntare all'IP del termostato WiFi e lo elenca nei device e nelle entità, anche se poi chiaramente non posso mapparlo su una card di tipo climate.
